scripts/publish.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+CPU_ARCHS="amd64 arm64 arm ppc64le s390x"
+TAG_COMMIT=$(git rev-list --abbrev-commit --tags --max-count=1)
+COMMIT=$(git rev-parse --short HEAD)
+TAG=$(git describe --abbrev=0 --tags ${TAG_COMMIT} 2>/dev/null || true)
```

Comment:
Why setting `|| true`? Do we expect `git describe` to be able to fail?

Reply:
eh, it's copy&paste, we can indeed remove this.